examples/heart.csv

```diff
@@ -0,0 +1,304 @@
+﻿age,sex,cp,trestbps,chol,fbs,restecg,thalach,exang,oldpeak,slope,ca,thal,target
```

Comment:
I think this was included by mistake since we are downloading it in the tutorial?

Reply:
yes. removed it.